types/project/CartConfiguration.raml

```diff
@@ -0,0 +1,7 @@
+#%RAML 1.0 DataType
+(package): Project
+type: object
+displayName: CartConfiguration
+properties:
+  countryTaxRateFallbackEnabled:
+    type: boolean
```

Comment:
Here it would be nice to copy the description of this field from the docs and add it as a `description:` property here. This would then add the description to our generated SDKs too.